OverlayPlugin.Core/MemoryProcessors/ClientFramework/ClientFramework.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace RainbowMage.OverlayPlugin.MemoryProcessors.ClientFramework
+{
+    public enum ClientLang
+    {
+        Japanese = 0,
+        English = 1,
+        German = 2,
+        French = 3,
+        // The global client only supports the above options, but presumably
+        // other clients have different byte values for their languages.
+        // We don't know what these are yet, so make all other possible
+        // values 'Unknown', and we can map if/once they are known.
+        Unknown4 = 4,
```

Comment:
`Chinese = 4,`

Just checked the 6.55 signature with CN client

Reply:
> `Chinese = 4,`
> 
> Just checked the 6.55 signature with CN client

Double-great!  Thanks for the info (and for confirming the 6.55 signatures work!)